includes/class-sensei-analysis.php

```diff
@@ -81,6 +84,74 @@ public function __get( $key ) {
 		}
 	}
 
+	/**
+	 * Add custom navigation to the admin pages.
+	 *
+	 * @since 4.2.0
+	 * @access private
+	 */
+	public function add_custom_navigation() {
+		// phpcs:ignore WordPress.Security.NonceVerification -- No action, nonce is not required.
+		$is_reports_page = isset( $_GET['page'] ) && ( $_GET['page'] === $this->page_slug );
```

Comment:
`$this->page_slug` should be replaced with the `PAGE_SLUG` constant.

If you didn't see the warning notice, maybe you should check if your `WP_DEBUG` is on?